vms/evm/subscriber.go

```diff
@@ -49,6 +50,8 @@ var warpFilterQuery = interfaces.FilterQuery{
 type subscriber struct {
 	nodeWSURL    string
 	nodeRPCURL   string
+	httpHeaders  map[string]string
+	queryParams  map[string]string
```

Comment:
We can maintain two `APIConfig` instances in this struct, one for the WS connection and one for the RPC connection.